adoc/chapters/programming_interface.adoc

```diff
@@ -12075,7 +12075,8 @@ is the cast to [code]#size_t#.
 a@
 [source]
 ----
-range operatorOP(const range& lhs, const size_t& rhs)
+// Available only when std::is_integral_v<T> is true
+range operatorOP(const range& lhs, const T& rhs)
```

Comment:
Here and elsewhere, I think you are missing the `template<typename T>`.

Reply:
Good catch!  Fixed in 73c3bc509c662f09931725c852a846530e7bd429.